src/os/win/pa_win_coinitialize.c

```diff
@@ -135,7 +135,7 @@ void PaWinUtil_CoUninitialize( PaHostApiTypeId hostApiType, PaWinUtilComInitiali
         DWORD currentThreadId = GetCurrentThreadId();
 		if( comInitializationResult->initializingThreadId != currentThreadId )
 		{
-			PA_DEBUG(("ERROR: failed PaWinUtil_CoUninitialize calling thread[%d] does not match initializing thread[%d]\n",
+			PA_DEBUG(("ERROR: failed PaWinUtil_CoUninitialize calling thread[%u] does not match initializing thread[%u]\n",
```

Comment:
`DWORD` is an unsigned long:

https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/262627d8-3418-4627-9218-4ffe110850b2

Most of the references that I have seen suggest to use `%lu`, for example:

https://www.davekb.com/browse_programming_tips:win32_printf_a_dword:txt
https://stackoverflow.com/questions/33671758/dword-format-specifier